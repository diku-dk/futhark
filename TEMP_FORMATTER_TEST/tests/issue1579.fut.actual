7737da230203816fb0f65377efc08e6b
