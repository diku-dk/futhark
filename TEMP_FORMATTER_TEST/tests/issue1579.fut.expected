f06a42c4061e15639b1c877a9fcbbafd
